web/models/summary_helper.ex

```diff
@@ -0,0 +1,80 @@
+defmodule CoursePlanner.SummaryHelper do
+  @moduledoc """
+    Provides helper functions for the summary page
+  """
+  use CoursePlanner.Web, :model
+
+  alias CoursePlanner.{Repo, Terms.Term, OfferedCourse}
+
+  def get_term_offered_course_for_user(user_id, user_role) do
```

Comment:
basically @laszlokardinal-dina want to call this from the controller. but we have the ID and role in the controller so I wrote it this way ( which i agree is not the best )